### DISCUSSION TITLE
Converting the export to a Website

### DISCUSSION THREAD

**Discussion by fezk21** [Jisco/VisualFamilyTree#24] [Q&A] [answered]
Hey, I am not well versed in coding, but I am trying to create the family tree in the editor and then take the export and convert it into a website to share with my family. The editor is so great because I can include the notes and everything on each person and I would want that to be saved on the website. I envision that the website looks the same as what the editor shows and then when you cntrl+click on each person the whole background on them shows. 

If anyone could help me with this or even give me a starting point that would be awesome. Thanks!

**Comment by Jisco** [comment, marked as answer]
Hi,
you can try something like [JsonCrack](https://github.com/AykutSarac/jsoncrack.com) (first result when i google "json to graph") to visualize the exported data when you dont want to code by yourself.

Good luck with your project, would be nice if you keep me posted on your progress 😃
